tests/tracer.go

```diff
@@ -41,8 +42,8 @@ func lostCb(count uint64) {
 }
 
 func main() {
-	if len(os.Args) != 1 {
-		fmt.Fprintf(os.Stderr, "Usage: %s\n", os.Args[0])
+	if len(os.Args) > 2 {
+		fmt.Fprintf(os.Stderr, "Usage: %s pid\n", os.Args[0])
```

Comment:
I refactored the code using Golang's flag so I don't have to write the "usage" myself.